misk/src/main/kotlin/misk/eventrouter/dependencies.kt

```diff
@@ -23,3 +38,8 @@ internal interface TopicPeer {
   fun acceptWebSocket(webSocket: WebSocket): WebSocketListener
   fun clusterChanged(clusterSnapshot: ClusterSnapshot)
 }
+
+fun shortName(host: String): String {
```

Comment:
not sure if we really need to keep this. If we do we should make it nicer

was handy for debugging